imgui/src/style.rs

```diff
@@ -153,19 +153,22 @@ unsafe impl RawCast<sys::ImGuiStyle> for Style {}
 
 impl Style {
     /// Scales all sizes in the style
+    #[doc(alias = "ScaleAllSizes")]
     pub fn scale_all_sizes(&mut self, scale_factor: f32) {
         unsafe {
             sys::ImGuiStyle_ScaleAllSizes(self.raw_mut(), scale_factor);
```

Comment:
Why do we bind these differently? This looks right though

Reply:
Are you referring to these alises?
```rust
    #[doc(alias = "StyleColors", alias = "StlyeColorsClassic")]
```
If so, it was because aliases are searched with (case-insensitive) exact match. But such aliases don't work well after all, so I'll remove them.

Reply:
Oh no, that's all good! lol no I shouldn't have just been musing -- these are good. Sorry lol, even just *reviewing* all of this my mind started wandering. That you wrote them all was impressive.